darts/utils/data/simple_inference_dataset.py

```diff
@@ -92,14 +100,19 @@ def __getitem__(self, idx: int) -> Tuple[TimeSeries, Optional[TimeSeries], Optio
             if self.n > self.output_chunk_length:
```

Comment:
Yep, but so far that will be automatically true since the only recurrent model so far is `RNNModel` which cannot have a `output_chunk_length` different from 1. But I added the check for future recurrent models.